packages/build-info/src/frameworks/nuxt.test.ts

```diff
@@ -51,3 +52,22 @@ describe('Nuxt V3', () => {
     expect(settings.clearPublishDirectory).toBeTruthy()
   })
 })
+
+describe('Nuxt V4', () => {
```

Comment:
just like the source, let's FLIP this. test the happy path (cases like v3 and v4 and `nuxt3` that result in the default config) in the main table test. and add a special case test below just for the legacy case(s).